resources.tf

```diff
@@ -207,6 +327,100 @@ data "aws_iam_policy_document" "analytical_dataset_write_s3" {
   }
 }
 
+resource "aws_iam_role_policy_attachment" "amazon_ssm_managed_instance_core" {
```

Comment:
Can this not be added to the current iam role?

Reply:
done